wpilibj/src/main/java/edu/wpi/first/wpilibj/drive/MecanumDrive.java

```diff
@@ -36,13 +36,13 @@
  * <p>Each drive() function provides different inverse kinematic relations for a Mecanum drive
  * robot.
  *
- * <p>This library uses the NED axes convention (North-East-Down as external reference in the world
- * frame): http://www.nuclearprojects.com/ins/images/axis_big.png.
- *
- * <p>The positive X axis points ahead, the positive Y axis points right, and the positive Z axis
+ * <p>The positive Y axis points ahead, the positive X axis points right, and the positive Z axis
  * points down. Rotations follow the right-hand rule, so clockwise rotation around the Z axis is
```

Comment:
With the doc updates, the coordinate axes no longer follow the right-hand rule. Rotations are just arbitrarily defined to be clockwise positive.